cluster/saltbase/pillar/README.md

```diff
@@ -0,0 +1,4 @@
+The
+[SaltStack pillar](http://docs.saltstack.com/en/latest/topics/pillar/)
+data is partially statically dervied from the contents of this
+directory, but largely generated at cluster inception today.
```

Comment:
- how is it generated at cluster inception?
- where can we go to find details on what goes in it?